syntax/ps1.vim

```diff
@@ -53,35 +55,47 @@ syn match ps1Constant +\$^+
 " Keywords reserved for future use
 syn keyword ps1Keyword class define from using var
 
-" Functions and Cmdlets
-syn match ps1Cmdlet /\w\+-\w\+/
```

Comment:
You could add `Function` highlighting to [your custom color scheme](https://github.com/heaths/vimfiles/blob/master/colors/custom.vim) with something like:

    hi Function ctermfg=3 cterm=bold guifg=#c0c000 gui=bold